### PR TITLE
Refactor nav

### PR DESCRIPTION
**Pull request by max-voronov**
Hello everyone,

The main change in this PR is the conversion of the component to a functional component.
There are also some cleanups.

Please review :)

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Thanks @max-voronov, this is much cleaner.

File: src/components/Nav.js [hunk 1]
```diff
@@ -1,85 +1,71 @@
...
-        <div className="site-nav-left" />
-        <div className="site-nav-middle">
-          <a
-            className="site-nav-logo"
-            href="https://coderadio.freecodecamp.org/"
+
+  const links = [
+    { href: "https://www.freecodecamp.org/news/", text: "News" },
+    { href: "https://www.freecodecamp.org/forum/", text: "Forum" },
+    { href: "https://www.freecodecamp.org/learn/", text: "Learn" }
```
Comment (RandellDawson) [inline src/components/Nav.js:14]: @ojeytonwilliams I thought we were moving away from hardcoding.  Shouldn't we use a config file of some sort?

**Comment by max-voronov** [comment]
> LGTM 👍
> 
> Thanks @max-voronov, this is much cleaner.

Thanks for the review!

File: src/components/Nav.js (hunk 1 above)
Comment (ojeytonwilliams) [reply src/components/Nav.js:14]: Generally speaking, yes, but this PR reduces duplication and overall makes the code easier to read, so I'm happy with it.

We'd want to pull things from a config file if we were creating multiple versions of the site with different links and text (if we translated Coderadio, say). At the moment it doesn't seem necessary.

**Pull request merged**